### PR TITLE
Mangasail extension

### PR DESCRIPTION
**Pull request by SnakeDoc83**
Filling request: https://github.com/inorichi/tachiyomi-extensions/issues/1211

### CODE REVIEW

File: src/en/mangasail/build.gradle [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ext {
+    appName = 'Tachiyomi: Mangasail'
+    pkgNameSuffix = 'en.mangasail'
+    extClass = '.Mangasail'
+    extVersionCode = 1
+    libVersion = '1.2'
+}
+
+dependencies {
+    implementation 'org.apache.commons:commons-text:1.6'
```
Comment (arkon) [inline src/en/mangasail/build.gradle]: I'd prefer if you didn't need this. You could use kotson for parsing JSON?

**Comment by arkon** [comment]
You should probably learn how to use Git rather than uploading files through GitHubt too.

**Comment by CarlosEsco** [comment]
Yea the commit history needs cleaned up for sure

File: src/en/mangasail/build.gradle (hunk 1 above)
Comment (SnakeDoc83) [reply src/en/mangasail/build.gradle]: The particular JSON string I was using that library for would sometimes have escaped characters that I didn't think I could solve with kotson.  Figured out a way after giving it some thought.  I won't need that library anymore.

**Comment by SnakeDoc83** [comment]
Going to resubmit after cleaning up.

**Comment by rama982** [comment]
Actually you can force push it without closing this pull.

**Pull request closed without merging**